packages/engineer/test/dev-server.unit.ts

```diff
@@ -586,6 +600,22 @@ describe('engineer:dev-server', function () {
         expect(pid1).to.eq(pid2);
         expect(pid1).to.eq(process.pid);
     });
+
+    it('runs app with the correct runtime metadata', async () => {
+        const {
+            outputPath,
+            config: { port },
+        } = await setup({
+            basePath: engineRuntimeMetadataFixturePath,
+            featureName: 'engine-runtime-metadata/x',
+        });
+
+        const page = await loadPage(`http://localhost:${port}/main.html`);
+        const text = await getBodyContent(page);
+
+        expect(text).to.contain(`"devport":${port}`);
```

Comment:
since the text is a stringified json, we can parse it ans make a test

Reply:
Sounds good, done